src/cmd_line/commands/write.ts

```diff
@@ -37,7 +37,7 @@ export class WriteCommand extends node.CommandBase {
     return this._arguments;
   }
 
-  execute(modeHandler : ModeHandler) : void {
+  async execute(modeHandler : ModeHandler) : Promise<void> {
```

Comment:
i need it to return the encapsulating promise from the `activeTextEditor.document.save()` call in the save method. otherwise i wasn't able to call `write` cmd and `quit` cmd synchronously.